### ISSUE TITLE
[Feature request] Add Voice variable: {GNameNext} and toggle voice: "Rest" in Count

### ISSUE THREAD

**Issue by a10n-jsd** [timer-machine/timer-machine-android#94]
Thank you very much for providing this app. This is the best timer apps for sport activities.

![github-count   rest](https://github.com/timer-machine/timer-machine-android/assets/74669521/b89e4d4e-993b-4844-b796-1ebd4711e773)

I wonder if I can make a notifier before the group using Next Group. There is only Next Step variable. 

For count: Is it possible to have a word "Rest" after count? It should have a toggle button i believe for user who want to make sound i.e. "3, 2, 1, Rest" instead of make a notifier for word "Rest" only. So, We can make a notifier after group with Name of Next Group.

That's it. I use this great app for run's training and it works smoothly. Much obliged

**Comment by DeweyReed** [comment]
Thanks for your suggestion!

- Should {GNameNext} be the name of the next group? If so, no problem.
- Please elaborate more on your first sentence.
- Adding a "Rest" voice in the last second of counting down can be unpredictable. Due to an unstable network or different languages' pronunciation, the speaking content could be longer than one second, and the speaking will be cut.

**Comment by a10n-jsd** [comment]
> Should {GNameNext} be the name of the next group? 

Yes. We can then make a notifier contains name of next group before the group. 

> Please elaborate more on your first sentence.

Without {GNameNext}, I have to create things:
1. A notifier, before the group, contains voice: next group (Hard writing)
2. A normal step with voice {GName} 
3. A notifier contains voice: next step and word "rest"

With {GNameNext}, It will simplify like this:
1. A notifier, before the group, contains voice: next group variable {GNameNext}
2. A normal step with voice of current step (let it be an empty content)
3. A notifier contains voice: rest only (Hard writing)


> Adding a "Rest" voice in the last second of counting down can be unpredictable. Due to an unstable network or different languages' pronunciation

Well, I thougth you don't have a plan for providing translation of Text-to-speech. In that case, please provide {Rest} variable so we don't have to "hard-writing" for Rest word only. 

You and Your app is very helpfull. Thank you very much indeed.

**Comment by DeweyReed** [comment]
You can mix plain text and variables in the content of a voice reminder. For example, "Next group {GName} rest."

A new variable {Rest} doesn't solve any problem. The app still needs to translate the word into different languages. Since there are so many languages, this approach is impractical.

**Comment by a10n-jsd** [comment]
> Should {GNameNext} be the name of the next group? If so, no problem.

To sum up, please accept our request for new variable: {GNameNext}

> A new variable {Rest} doesn't solve any problem

Yes, you're right. {Rest} variable is impractical and doesn't solve the problem I mentioned earlier.

**Comment by 2018robert** [comment]
+1
